chainer/serializers/npz.py

```diff
@@ -94,7 +94,9 @@ def __getitem__(self, key):
     def __call__(self, key, value):
         key = key.lstrip('/')
         dataset = self.npz[self.path + key]
-        if isinstance(value, numpy.ndarray):
+        if value is None:
```

Comment:
Yes, it looks like I should add the check for `value` is `None` to `HDF5Deserializer` as well. In that case, I would agree that we should fix the documentation of `AbstractSerializer` to mention that for deserializers, `value` can also be `None`.